### PR TITLE
chore: additional vjs ad errors

### PR DESCRIPTION
**Pull request by wseymour15**
## Description
Add additional errors to the video.js error interface regarding ads.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Review by dzianis-dashkevich — APPROVED** [review]

File: src/js/consts/errors.js [hunk 1]
```diff
@@ -9,5 +9,12 @@ export default {
...
   VttLoadError: 'vtt-load-error',
   VttCueParsingError: 'vtt-cue-parsing-error',
-  EMEKeySessionCreationError: 'eme-key-session-creation-error'
+  EMEKeySessionCreationError: 'eme-key-session-creation-error',
+  // Errors used in contrib-ads:
+  AdsBeforePrerollError: 'ads-before-preroll-error',
+  AdsPrerollError: 'ads-preroll-error',
+  AdsMidrollError: 'ads-midroll-error',
+  AdsPostrollError: 'ads-postroll-error',
+  AdsMacroReplacementFailed: 'ads-marco-replacement-failed',
```
[the remark this answers is not in the document]
Comment (wseymour15) [reply src/js/consts/errors.js:18]: Good eye, PR for that update: https://github.com/videojs/video.js/pull/8628

It is not an urgent fix, so no need to rush a release for this one.

**Comment by thijstriemstra** [comment]
Interesting to see this being added, since it's a plugin/thirdparty library afaik.

**Comment by misteroneill** [comment]
> Interesting to see this being added, since it's a plugin/thirdparty library afaik (and shouldn't be "bloating" a core video.js).

Yeah, we agree that it's not the ideal place for these constants, but was added in the interest of the simplicity of having a single mapping of error codes in Video.js and it had a fairly minimal impact on bundle size.

**Comment by thijstriemstra** [comment]
Thanks for the feedback.

>  in the interest of the simplicity 

is always a red flag isn't it.

**Pull request merged**